client/my-sites/comments/main.jsx

```diff
@@ -47,12 +52,33 @@ export class CommentsManagement extends Component {
 			translate,
 		} = this.props;
 
+		const jetpackUpdateLink = `/plugins/jetpack/${ this.props.siteSlug }`;
+
+		const showJetpackUpdateScreen =
```

Comment:
Couldn't we create this directly in `mapStateToProps`?

Reply:
Yup, makes sense and I'll update it.

Reply:
Updated in 5676b5222406d81fea2771233224456b58406758.